tests/unit_grpc/test_grpc_index_upsert.py

```diff
@@ -19,9 +19,27 @@ class MockUpsertDelegate:
     def __init__(self, upsert_response: UpsertResponse):
         self.response = upsert_response
 
-    def result(self, timeout):
+    def result(self, timeout=None):
```

Comment:
Just OOC, did you set `timeout` to `None` here b/c you automatically set it to `5s` if it's `None` in  `future.py`?

Reply:
This `MockUpsertDelegate` is a standin for the grpc future returned from the grpc runner. That object has this interface: https://grpc.github.io/grpc/python/_modules/grpc.html#Future.result

I adjusted the signature to match the `result` args in the interface.

Reply:
ah thanks!